tests/net/webrequest.cpp

```diff
@@ -263,7 +263,7 @@ TEST_CASE_METHOD(RequestFixture,
     pos += strlen(expectedKey);
 
     // There may, or not, be a space after it.
-    while ( wxIsspace(response[pos]) )
+    while ( wxIsspace(response[pos]) || response[pos] == '[' )
```

Comment:
Could you please update the comment to explain why do we need to test for `[` here? I don't really understand it...